modules/optimizing-mtu-networking.adoc

```diff
@@ -9,7 +9,7 @@ There are two important maximum transmission units (MTUs): the network interface
 
 The NIC MTU is only configured at the time of {product-title} installation. The MTU must be less than or equal to the maximum supported value of the NIC of your network. If you are optimizing for throughput, choose the largest possible value. If you are optimizing for lowest latency, choose a lower value.
 
-The network plugin overlay MTU must be less than the NIC MTU by 50 bytes at a minimum. This accounts for the SDN overlay header. So, on a normal ethernet network, set this to `1450`. On a jumbo frame ethernet network, set this to `8950`.
+The network plugin overlay MTU must be less than the NIC MTU by 50 bytes at a minimum. This accounts for the SDN overlay header. So, on a normal ethernet network, this should be set to `1450`. On a jumbo frame ethernet network, this should be set to `8950`. These values should be set automatically by the Cluster Network Operator based on the NIC's configured MTU. Therefore, cluster administrators do not typically update these values. Amazon Web Services (AWS) and bare-metal environments support jumbo frame ethernet networks. This setting will help throughput, especially with transmission control protocol (TCP).
```

Comment:
@ahardin-rh Looks OK to me.

I wonder if we can't clarify that this 50-bytes-less is only for openshift-sdn though; it's a bit confusing to have "the network plugin overlay MTU"; perhaps we should change that to "The openshift-sdn network plugin overlay MTU"?

Reply:
Updated. Thanks, Dan!